src/lm_buddy/jobs/evaluation/hf_evaluate.py

```diff
@@ -0,0 +1,115 @@
+"""
+lm-buddy entrypoint to run summary evaluation using huggingface eval
+"""
+
+import json
+import time
+from pathlib import Path
+
+import s3fs
+from loguru import logger
+from tqdm import tqdm
+
+from lm_buddy.configs.jobs.hf_evaluate import HuggingFaceEvalJobConfig
+from lm_buddy.configs.vllm import VLLMCompletionsConfig
+from lm_buddy.constants import LM_BUDDY_RESULTS_PATH
+from lm_buddy.jobs.asset_loader import (
+    HuggingFaceDatasetLoader,
+    HuggingFaceModelLoader,
+)
+from lm_buddy.jobs.common import EvaluationResult
+from lm_buddy.jobs.evaluation.metrics import EvaluationMetrics
+from lm_buddy.jobs.model_clients import (
+    HuggingFaceModelClient,
+    OpenAIModelClient,
+    PipelineModelClient,
+)
+
+
+def save_outputs(config: HuggingFaceEvalJobConfig, evaluation_results: dict) -> Path:
+    storage_path = config.evaluation.storage_path
```

Comment:
We might have two different functions here to save locally and save to the cloud and abstract away into a try/except, i.e. 

```
def save_to_disk():
      ```save local output```

def save_to_s3():
      ```save to s3 output```
      
try:
   save_local()
   save_cloud()
except:
   logger.info("Wasn't able to save")
 ```

Reply:
Yeah that would be cleaner - I already miss our artifacts so much! Are we sure we don't want mlflow to manage all of this? 😅